mithril-aggregator/src/event_store/transmitter_service.rs

```diff
@@ -36,31 +35,10 @@ where
 }
 
 impl TransmitterService<EventMessage> {
-    /// Craft and send an [EventMessage] given the serializable data.
-    /// This method is done in a way to make as simple as possible to send a
-    /// message and make any error not to cause a business failure. A warning is
+    /// Send an [EventMessage].
+    /// This method make any error not to cause a business failure. A warning is
     /// issued so the resulting error may be discarded.
```

Comment:
```suggestion
    /// Send an [EventMessage].
    ///
    /// A warning is logged when an error is raised so callers may safely ignore the
    /// returned error to avoid business failure.
```